src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -69,7 +61,8 @@ const UsersFragment = ({
       data: {
         user_id: myId,
         other_id: item.user_id,
-        username: item.username
+        username: item.username,
+        following: item.following
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding the `following` field to the data object seems to be a good addition, assuming that the backend service can handle this additional field.



<!-- This is an auto-generated comment by OSS CodeRabbit -->